homeassistant/util/color.py

```diff
@@ -1,5 +1,7 @@
 """Color util methods."""
 import math
+# pylint: disable=import-error
+import webcolors
```

Comment:
We should just import the functions from webcolors that we want to expose, no need to create wrappers.

```python
from webcolors import html5_parse_legacy_color as color_name_to_rgb
```